src/components/routes/Host/HostListingCard/HostListingCard.tsx

```diff
@@ -79,12 +80,18 @@ const HostListingCard = (props: Props): JSX.Element => {
           <div className='host-listing-publish'>
             <label htmlFor={`publish-${id}`} title={canPublish ? '' : INCOMPLETE_LISTING}>
               <span className={canPublish ? '' : 'host-listing-meta--disabled'}>Publish</span>
-              <Switch
-                checked={isActive}
-                disabled={!canPublish}
-                onColor={hexColor('correct')}
-                onChange={() => toggleListing(id).catch((error) => alert(error))}
-                id={`publish-${id}`} />
+              <FirebaseConsumer>
+                {({ completedVerification }: FirebaseUserProps) => {
+                  return (
+                    <Switch
+                      checked={isActive}
+                      disabled={!canPublish || !completedVerification}
+                      onColor={hexColor('correct')}
+                      onChange={() => toggleListing(id).catch((error) => alert(error))}
+                      id={`publish-${id}`} />
+                  );
+                }}
+              </FirebaseConsumer>
             </label>
             {!canPublish && <p className='host-listing-notice'>{INCOMPLETE_LISTING}</p>}
```

Comment:
maybe we can do ``!completedVerification ^^ "You must verify your email and phone to publish."``

Reply:
👍